app/src/main/res/layout/fragment_setting.xml

```diff
@@ -1,13 +1,151 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/colorPrimaryDark">
+    xmlns:app="http://schemas.android.com/apk/res-auto">
 
```

Comment:
Should we let users delete their account eventually? Future feature to consider. Overall I approve this pull request as is.

Reply:
I'll work on this on a separate branch.